python_errors/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
python_errors/config.py,sha256=2vnAllIqiXGhdaxl51g_n2EkFhXxm_jomeGwfHKpDow,2101
python_errors/handle_traceback.py,sha256=r83NwGDRwOSjxoCD0TYvOo5R2HZm_aEVbXyghV8nm7c,1250
python_errors/secure.py,sha256=t8lqYzD4kUQGLZZbrhmxdKoCzI99zTWaPArOvu9ihoo,13403
python_errors-0.0.5.dist-info/LICENSE,sha256=RpOWq4cnvrgRkNdtLgni79rLd3lyUVjKzKDVBVPmjYM,1086
python_errors-0.0.5.dist-info/METADATA,sha256=GxnXrGGNw_MwV4NmIdlg7hbozt8Trds13c8SPcjkaDM,4699
python_errors-0.0.5.dist-info/WHEEL,sha256=PZUExdf71Ui_so67QXpySuHtCi3-J3wvF4ORK6k_S8U,91
python_errors-0.0.5.dist-info/top_level.txt,sha256=XXwgrmZlawKf_Yzrc25TbRn18Rq8kyWqS91t-OaTDVM,14
python_errors-0.0.5.dist-info/RECORD,,
